tests/libcxx/expected_results.txt

```diff
@@ -712,6 +712,11 @@ std/iterators/iterator.requirements/iterator.concepts/iterator.concept.random.ac
 # Bogus test expects to_address() to SFINAE away for int.
 std/utilities/memory/pointer.conversion/to_address_without_pointer_traits.pass.cpp FAIL
 
+# libc++'s "ascii" mode considers every code unit to represent a valid printable narrow character.
+# We don't support such a mode.
+std/utilities/format/format.functions/ascii.pass.cpp FAIL
+std/utilities/format/format.functions/escaped_output.ascii.pass.cpp FAIL

```

Comment:
`FAIL` feels wrong for this: I don't care that these are failing and wouldn't care if they start passing. Let's `SKIPPED` them instead.